charts/index.yaml

```diff
@@ -3,7 +3,29 @@ entries:
   karmada:
   - apiVersion: v2
     appVersion: latest
-    created: "2023-08-03T16:11:27.348187800+08:00"
+    created: "2023-09-11T14:40:33.2057800-04:00"
+    dependencies:
+    - name: common
+      repository: https://charts.bitnami.com/bitnami
+      version: 1.x.x
+    description: A Helm chart for karmada
+    digest: 93a7051ab973c103411f74d7a516c673f64db447a4cdb1eae98efafff93a48b1
+    kubeVersion: '>= 1.16.0-0'
+    maintainers:
+    - email: jrkeen@hotmail.com
+      name: jrkeen
+    - email: jackson.cloudnative@gmail.com
+      name: pidb
+    - email: shentiecheng@huawei.com
+      name: Poor12
+    name: karmada
+    type: application
+    urls:
+    - https://github.com/karmada-io/karmada/releases/download/charts/karmada-chart-v1.7.0.tgz
```

Comment:
Hello, I think we should use `helm repo index charts/karmada --url https://github.com/karmada-io/karmada/releases/download/v1.7.0 --merge charts/index.yaml`

otherwise the urls is not valid

you can see this `https://github.com/karmada-io/karmada/releases/download/charts/karmada-chart-v1.7.0.tgz` is 404~